content/general/limits.textile

```diff
@@ -347,17 +391,38 @@ h3(#concurrent-functions). Function concurrency
 
 Function concurrency is the maximum number of functions that can run at the same time. This rate only applies to Google Cloud functions and Azure functions. AWS Lambda functions do not count towards this rate as they use the asynchronous @event@ invocation type.
 
+h3(#integration-warnings). Integration warnings
+
+Warnings for integrations are sorted into integration type and then warning type:
+
+* @reactor.httpEvent.maxRate.hard@ - when the hard limit on the account has been exceeded.
```

Comment:
In fact, for these global aggregate rate limits, the warning rate is 50%, not 80%, as with other hard limits.

Reply:
@StevenAppleby - FYI